experimental/Schemes/Types.jl

```diff
@@ -364,9 +362,9 @@ identifications given by the glueings in the `default_covering`.
         G = default_covering(X)(ambient_scheme(V), ambient_scheme(W))
         f, g = glueing_morphisms(G)
         VG = intersect(V, domain(f))
-        preV = preimage(g, VG)
+        preV = preimage(g, VG, check=false)
```

Comment:
This line is unreached, as the whole function is deactivated in line 352. Is this intentional and do you intentionally do changes in this unreached part?